software/local_firesim/load_generator/README.md

```diff
@@ -0,0 +1,7 @@
+Setup instructions:
+
+This is modeled after local firesim. Follow the setup instructions in README.md in the local_firesim directory in order to use the load generator.
+
+Notes:
+
+This load generator is based on the software switch in chipyard/sims/firesim/target-design/switch, but uses unique files. Unlike local firesim, which contains symbolic links to the actual firesim switch (and should therefore be nearly identical to it aside from switchconfig.h), this load generator contains copies of the original files that will gradually diverge over time. This means that, for now, the load generator will work in local firesim only, and not in the full AWS firesim.
```

Comment:
Is this still accurate?

Reply:
For the copy of the load generator here in local_firesim, yes it's still accurate. Eventually I'd like to get rid of this one and just have the local_firesim scripts use the switch, now that everything is integrated into that, but I think it might be easier to do that in a separate PR later.